tensorflow/core/common_runtime/process_util.cc

```diff
@@ -154,8 +154,12 @@ int32 NumInterOpThreadsFromSessionOptions(const SessionOptions& options) {
 }
 
 thread::ThreadPool* NewThreadPoolFromSessionOptions(
-    const SessionOptions& options) {
-  const int32_t num_threads = NumInterOpThreadsFromSessionOptions(options);
+    const SessionOptions& options, bool run_in_caller) {
```

Comment:
Rather than a boolean, I suggest supplying the actual number of threads (or 0/negative if you want to read from session options).

Reply:
Done.